mindsdb/interfaces/knowledge_base/controller.py

```diff
@@ -88,6 +88,13 @@ def select_query(self, query: Select) -> pd.DataFrame:
         # send to vectordb
         db_handler = self.get_vector_db()
         resp = db_handler.query(query)
+        new_columns = [
+            self._kb.params.get('id_column', TableField.ID.value),
+            self._kb.params.get('content_columns', TableField.CONTENT.value),
+            self._kb.params.get('metadata_columns', TableField.METADATA.value)
+        ]
+
+        resp.data_frame.columns = new_columns
```

Comment:
is better not use it, the number of columns could be different. and output column names should be already correct

Reply:
ok, but i think id_column need to be reflected in output